package.json

```diff
@@ -33,17 +37,17 @@
     "scripts": {
         "autobuild": "rollup -c -w",
         "build": "rollup -c",
-        "dev": "karma start --auto-watch --no-single-run --browsers chrome --grep",
-        "dev:ff": "karma start --auto-watch --no-single-run --browsers firefox --grep",
+        "dev": "karma start ./karma.conf.cjs --auto-watch --no-single-run --browsers chrome --grep",
+        "dev:ff": "karma start ./karma.conf.cjs --auto-watch --no-single-run --browsers firefox --grep",
         "docs": "npm run build && vuepress build docs --no-cache",
         "docs:dev": "npm run build && vuepress dev docs --no-cache",
         "lint-js": "eslint \"src/**/*.js\" \"test/**/*.js\" \"docs/**/*.js\"",
         "lint-md": "eslint \"**/*.md\"",
         "lint-tsc": "tsc",
-        "lint-types": "eslint \"types/**/*.ts\" && node -r esm types/tests/autogen.js && tsc -p types/tests/",
+        "lint-types": "eslint \"types/**/*.ts\" && npm run build && node types/tests/autogen.js && tsc -p types/tests/",
```

Comment:
Why does the autogen.js depend on build now?

Reply:
@kurkle 

https://github.com/chartjs/Chart.js/pull/10525/files#diff-a1e51d07b802d38e32299338867451d955fe5078a06398bb20a7d0aef4d03a23

Native ESM requires extenstion in import paths, so we can't import helpers from the sources. Or we should add `.js` extenstion in every import in src.

Reply:
I'm not sure its worth the effort adding the .js to every import, considering the possible .ts change.